flink-formats/flink-avro/src/main/java/org/apache/flink/formats/avro/typeutils/AvroSchemaConverter.java

```diff
@@ -297,10 +297,10 @@ private static DataType convertToDataType(Schema schema) {
 	 * @return Avro's {@link Schema} matching this logical type.
 	 */
 	public static Schema convertToSchema(LogicalType logicalType) {
-		return convertToSchema(logicalType, 0);
+		return convertToSchema(logicalType, "row_0");
```

Comment:
I think this will conflict if a field name is "row_0".

Reply:
Yeah, I think we can use "row_" append the current timestamp as default row name. Do you have any good idea?